widgetschema.json

```diff
@@ -130,21 +134,50 @@
             "oneOf": [
                 {
                     "properties": { "type": { "const": "inputGroup" } },
-                    "required": ["name", "type", "inputGroup"]
+                    "required": ["name", "type", "inputGroup"],
+                    "exclude": ["fields", "itemDisplayName"]
                 },
-                { "properties": { "type": { "const": "actionButtons" } } },
-                { "properties": { "type": { "const": "documentUpload" } } },
-                { "properties": { "type": { "const": "columnSelector" } } },
-                { "properties": { "type": { "const": "filter" } } },
                 {
-                    "properties": { "type": { "const": "text" } }
+                    "properties": { "type": { "const": "text" } },
+                    "exclude": ["fields", "itemDisplayName", "inputGroup"]
+                },
+                {
+                    "properties": { "type": { "const": "number" } },
+                    "exclude": ["fields", "itemDisplayName", "inputGroup"]
+                },
+                {
+                    "properties": { "type": { "const": "date" } },
+                    "exclude": ["fields", "itemDisplayName", "inputGroup"]
+                },
+                {
+                    "properties": { "type": { "const": "boolean" } },
+                    "exclude": ["fields", "itemDisplayName", "inputGroup"]
                 },
-                { "properties": { "type": { "const": "number" } } },
-                { "properties": { "type": { "const": "date" } } },
-                { "properties": { "type": { "const": "boolean" } } },
                 {
                     "properties": { "type": { "const": "choices" } },
-                    "oneOf": [{ "required": ["name", "type", "choices"] }, { "required": ["name", "type", "api"] }]
+                    "oneOf": [{ "required": ["choices"] }, { "required": ["api"] }],
+                    "exclude": ["fields", "itemDisplayName", "inputGroup"]
+                },
+                {
+                    "properties": { "type": { "const": "multiple" } },
+                    "required": ["fields", "itemDisplayName"],
```

Comment:
I guess I still don't think `itemDisplayName` should be required.  Does something not work if there is no `itemDisplayName`?